packages/strapi-design-system/src/Combobox/__tests__/utils.spec.js

```diff
@@ -6,9 +6,48 @@ describe('Combobox | utils', () => {
       const options = [{ props: { children: 'Hamburger' } }, { props: { children: 'Bagel' } }];
       const actual = filterOptions(options, 'ham');
       const expected = [{ props: { children: 'Hamburger' } }];
+
       expect(actual).toEqual(expected);
     });
 
+    describe('should filter the options correctly, with partial term', () => {
+      const options = [
+        { props: { children: 'hicham/project-super-cms' } },
+        { props: { children: 'pierre/project-awesome-cms' } },
+        { props: { children: 'udi/cool-cms' } },
+        { props: { children: 'strapi/strapi' } },
+        { props: { children: 'strapi/design-system' } },
+      ];
+
+      it('3 of 5 items', () => {
+        const result = filterOptions(options, 'cms');
+        const expected = [options[0], options[1], options[2]];
+
+        expect(result).toEqual(expected);
+      });
+
+      it('2 of 5 items', () => {
+        const result = filterOptions(options, 'project');
+        const expected = [options[0], options[1]];
+
+        expect(result).toEqual(expected);
+      });
+
+      it('2 last of 5 items', () => {
+        const result = filterOptions(options, 'strapi');
+        const expected = [options[3], options[4]];
+
+        expect(result).toEqual(expected);
+      });
+
+      it('1 item', () => {
```

Comment:
What do you think about adding one more test, to prove the edge-case `filterOptions(options, '')` or `filterOptions(options, null)`is working well?

Reply:
+1

Reply:
Cool.
Those test-cases already exists.
I've re-order them now, adding one more.
Thank you